### PR TITLE
gh-190 Refactor AbstractCoreKeyAccumuloElementConverter.java

### PR DESCRIPTION
**Pull request by GCHQDeveloper404**
Please use your finest toothed comb please.

### CODE REVIEW

**Comment by GCHQDeveloper404** [comment]
There will be more, I'm just doing this in byte sized pieces.

File: store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java [hunk 1]
```diff
@@ -291,41 +270,34 @@ public Properties getPropertiesFromColumnVisibility(final String group, final by
...
-                    } else {
-                        final byte[] bytes = serialiser.serialiseNull();
-                        writeBytes(bytes, out);
-                    }
-                } else {
-                    writeBytes(AccumuloStoreConstants.EMPTY_BYTES, out);
+            byte[] bytes = AccumuloStoreConstants.EMPTY_BYTES;
+            if (serialiser != null) {
+                Object value = properties.get(propertyName);
+                if (value != null) {
```
Comment (p013570) [inline store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java]: If value is null then we should use serialiser.serialiseNull()

Comment (GCHQDeveloper404) [inline store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java]: I shall implement a test for a serialiser that overwrites the default.

**Comment by GCHQDeveloper404** [comment]
Testing added for serialiser that overwrites the default serialiseNull()

**Pull request merged**